docs/090-breaking-changes.rst

```diff
@@ -9,7 +9,7 @@ For the full list check
 
 Silent Changes of the Semantics
 ===============================
-
+- Make basfee as a reserved identifier.
```

Comment:
It should go under "New restrictions" and please copy the suggested texted for changelog there.